BP/common/landed_titles/BP_e_byzantium.txt

```diff
@@ -8017,38 +7864,69 @@ e_byzantium = {
 					color2 = { 255 255 255 }
 				}
 			}
+			c_achaia = {
```

Comment:
Why are you changing the the de jure here?

Reply:
Corinth was the capital of Achaia province and it was the largest, most important city in the Peloponnese well into the Middle Ages. It was the site of the Hexamilion Wall as well (which I added as a Duchy building for Corinth).